mesonbuild/modules/gnome.py

```diff
@@ -969,12 +969,18 @@ def yelp(self, state, args, kwargs):
             raise MesonException('Yelp requires a project id')
 
         project_id = args[0]
+        if len(args) > 1:
+            FeatureDeprecated.single_use('gnome.yelp variadic source args', '0.60.0', 'use the "sources" keyword argument instead.')
```

Comment:
IMHO "variadic" is too technical, most user won't know what it is about. We already have tones of APIs where we say something along "more than one positional argument".